gulpfile.js

```diff
@@ -61,9 +61,19 @@ gulp.task('sass', function() {
 });
 
 gulp.task('watch', function() {
-  gulp.watch(src, ['lint']);
-  gulp.watch('assets/polymer.html', ['vulcanize']);
-  gulp.watch('assets/*.scss', ['sass']);
+    gulp.watch(src, gulp.series('lint'));
+    gulp.watch('assets/polymer.html', gulp.series('vulcanize'));
+    gulp.watch('assets/*.scss', gulp.series('sass'));
 });
 
-gulp.task('default', ['connect', 'sass', 'lint', 'test', 'watch']);
\ No newline at end of file
+gulp.task('default',
+    gulp.parallel('connect','sass', gulp.series( 'lint', 'test', 'watch'))
+);
+
+// var livingcss = require('gulp-livingcss');
```

Comment:
Are these comments still needed?

Reply:
no. I was testing the plugin in the same space. Apologies, I will should have seen those in the merge.